src/app/beer_garden/config.py

```diff
@@ -723,6 +723,95 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
     },
 }
 
+_PARENT_SPEC = {
+    "type": "dict",
+    "items": {
+        "http": {
+            "type": "dict",
+            "items": {
+                "enable": {
+                    "type": "bool",
+                    "default": False,
+                    "description": "Publish events to parent garden over HTTP",
+                    "alt_env_names": ["PARENT_HTTP"],
```

Comment:
I *guess* I can be ok with this. I'd prefer to just set the normal name `PARENT_HTTP_ENABLE` over adding an alt name that is the same as the parent_http config item. But if you think it makes a difference and it works then :shrug: